src/clients/third-party-service-client.ts

```diff
@@ -102,6 +103,13 @@ export const getThirdPartyServiceClient =
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         thirdPartyConfig.prodEnvironment ?? thirdPartyConfig.testEnvironment!;
 
+    eventLog.task.info(`Third party client base url: ${environment.baseUrl}`);
+    eventLog.task.info(
```

Comment:
...and do not remove this. I know that now the check is the same, but I would keep this for possibile future improvements.